kubelet-to-gcm/Dockerfile

```diff
@@ -12,15 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:14.04
+FROM scratch
 MAINTAINER Quintin Lee "qlee@google.com"
 
-# Disable prompts from apt.
-ENV DEBIAN_FRONTEND noninteractive
-
-# # Install the Fluentd agent that knows how to send logs to Google Cloud Logging.
-RUN apt-get -q update && \
-    apt-get install -y curl && \
-    apt-get clean
-
 ADD build/monitor monitor
```

Comment:
It looks like this image should work fine running as `nobody`. Mind adding a `USER` directive? See https://github.com/kubernetes/heapster/pull/1514/commits/c0f5ab124765c09c0d7a679e843d8ffd1aaca016 for an example.